src/main/java/com/rarchives/ripme/ripper/rippers/AerisdiesRipper.java

```diff
@@ -0,0 +1,98 @@
+package com.rarchives.ripme.ripper.rippers;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jsoup.Connection.Response;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import com.rarchives.ripme.ripper.AbstractHTMLRipper;
+import com.rarchives.ripme.ui.RipStatusMessage.STATUS;
+import com.rarchives.ripme.utils.Http;
+import java.util.HashMap;
+
+public class AerisdiesRipper extends AbstractHTMLRipper {
+
+    private Document albumDoc = null;
+    private Map<String,String> cookies = new HashMap<String,String>();
+
+
+    public AerisdiesRipper(URL url) throws IOException {
+        super(url);
+    }
+
+    @Override
+    public String getHost() {
+        return "aerisdies";
+    }
+    @Override
+    public String getDomain() {
+        return "aerisdies.com";
+    }
+
+    @Override
+    public String getGID(URL url) throws MalformedURLException {
+        Pattern p = Pattern.compile("^https?://www.aerisdies.com/html/lb/([a-z]*_[0-9]*_\\d)\\.html");
```

Comment:
Wouldn't most people copy and paste the link? Also wouldn't a link missing the l in html just 404? I just don't really see anything to gain by matching malformed/incomplete links